control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp

```diff
@@ -107,6 +107,14 @@ class TRAJECTORY_FOLLOWER_PUBLIC PidLongitudinalController : public Longitudinal
   // control state
   enum class ControlState { DRIVE = 0, STOPPING, STOPPED, EMERGENCY };
   ControlState m_control_state{ControlState::STOPPED};
+  std::string toStr(const ControlState s)
```

Comment:
reference

Reply:
add const to the function

"`toStr` of pid_longitudinal_controller" does not make sense to me. `convertControlStateToString` or `getStringControlState` sounds better.

Reply:
`toStr(enum)` still sounds odd? It receives the enum and return as a string. Sounds straightforward to me.

Reply:
Alright. This may be kind of preference, so if you think it's okay, forget my comment.

(I feel that
- "toStr of pid_longitudinal_controller class" does not make sense to me. We cannot understand what to convert to string only with the function name, until seeing the function's implementation.
- For example, "toStr of ControlState class" makes sense to me. We can almost understand what to convert to string only with the function name.

Reply:
I agree. Now the ControlState is implemented in the longitudinal_controller class as an enum, so the "toStr of ControlState class" can not be implemented. I hope an enum class could have methods.

>  We cannot understand what to convert to string only with the function name,

`toStr()` is definitely vague, but I feel, with an arg expression `toStr(controlState)` looks obvious for its role.

Reply:
Okay!